client/storybook/preview.js

```diff
@@ -2,10 +2,13 @@ import '../tests/stubs';
 
 import '../../wagtail/admin/static_src/wagtailadmin/scss/core.scss';
 import '../../wagtail/admin/static_src/wagtailadmin/scss/sidebar.scss';
+import './preview.scss';
+
+window.PATTERN_LIBRARY_API = '/pattern-library/api/v1/render-pattern';
```

Comment:
Is there any way to avoid this being a global window variable, I know it is in the context of storybook only but could create a risk of being written by other code accidentally and adds a bit of indirection.

Maybe it can be an exported const somewhere?

Reply:
I have refactored this to be hard-coded for now (it’s only used in two places). Will try to think of a better way to do this later on, as part of a new release of `storybook-django`.